radio/src/Makefile

```diff
@@ -1801,9 +1801,11 @@ clean_list :
 	$(REMOVE) bitmaps/*.lbm
 	$(REMOVE) bitmaps/*/*.lbm
 	$(REMOVE) bitmaps/*/*/*.lbm
+	$(REMOVE) bitmaps/Taranis/mainmenu.png
```

Comment:
This one should come in another commit, which removes the code I wrote for it, as it won't be useful anymore